File: seminar_4/Python/HW_task2.py
# Напишите программу, которая принимает на вход число и выдаёт сумму цифр в числе.
# 452 -> 11
# 82 -> 10
# 9012 -> 12

def SumNumbers(number):
    result = 0

    while number > 0:
        result += number % 10
        number //= 10

    return result


num = int(input("Введите число: "))
print(f"Сумма цифр в числе {num} - {SumNumbers(num)}")

File: seminar_4/Java/HW_task2.java
// Напишите программу, которая принимает на вход число и выдаёт сумму цифр в числе.
// 452 -> 11
// 82 -> 10
// 9012 -> 12
package seminar_4.Java;

import java.util.Scanner;

public class HW_task2 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Введите число: ");
        int num = input.nextInt();
        int count = 0;
        while (num > 0) {
            count += num % 10;
            num /= 10;
        }
        System.out.println(count);
    }
}
